src/pages/cli/project/troubleshooting.mdx

```diff
@@ -235,6 +235,22 @@ Amplify currently doesn't support automatic data migration. If the data in the o
 If you have previously deployed an application using GraphQL transformer V1 and have decided to migrate to GraphQL transformer V2, go through the [GraphQL transformer migration documentation](/cli/migration/transformer-migration/#changes-that-amplify-cli-will-auto-migrate-for-you) to understand the impact of the migration on the schema directives and limitations prior to updating.
 You can confirm the version of the GraphQL transformer using the `amplify status` command. You can also inspect the `transformerversion` feature flag in `${project-root}/amplify/cli.json`, to confirm.
 
+### Migration from Amplify version using AWS CDK V1 to V2
+If you have previously deployed an application based on Amplify version < 11.0.0 which is using overrides or custom resources, you can follow the below guide if you are in process of migration amplify version 10 to 11: 
```

Comment:
I suggest simplifying the wording a bit:

```suggestion
### Version 11 migration from AWS CDK V1 to V2
If you have deployed an application which is using overrides or custom resources prior to Amplify CLI version 11.0.0, follow the steps below to migrate:
```

Additionally, is there any CDK documentation we can point that detail any breaking changes? I see this migration guide is in troubleshooting, but I think it makes more sense to have a standalone entry in the `Migration & Backwards Compatibility` section of the docs.

Reply:
There is no migration steps involved for most of the users as amplify with `cdkv2` is almost a non breaking change.

I was confused to have it in migration section. This documentation helps when customers manually upgrade their package.json and `.ts` file to upgrade to `cdkv2` and  this link will be in the warning message if Cx are on `cdkV1`.


I have added `cdkv2` documentation at the end .